cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -398,20 +397,24 @@ impl PathEnvironment {
             ))?,
         }
 
-        let template_contents = fs::read_to_string(env!("MANIFEST_TEMPLATE"))
-            .map_err(EnvironmentError::ReadManifest)?;
-
-        let manifest_contents =
-            Self::replace_placeholders(&template_contents, system, customization);
+        // Read and parse `manifest.toml` TOML template asset containing placeholders
+        let mut manifest_template = fs::read_to_string(env!("MANIFEST_TEMPLATE"))
```

Comment:
note (not something you need to fix, just a thought): I'm not really sure why we're even reading this file instead of just using `include_str`. I think it might make more sense to use `include_str` inside the templating function, and then we can ensure with unit tests that the file can be parsed as TOML. And the templating function doesn't need to support arbitrary `&RawManifest`s, it just needs to support this one.

Is there any reason we're not using `include_str` @ysndr @zmitchell ?

Reply:
technically `fs::read_to_string(env!("MANIFEST_TEMPLATE"))` allows you to try different templates without recompiling. It also allows you to inspect the template separately from the binary


However, since the only case where you want to change these is during development where you can just `cargo run` and being able to _see_ the template is not really relevant (and the file is small) I think using `include_str` at least prevents a relatively rare runtime error from trying to read the file.